### ISSUE TITLE
Scipy.signal

### ISSUE THREAD

**Issue by juvenalandres** [xtyangpsp/SeisGo#31]
Hi again. The previous #29 error was resolved but I'm finding something new at the moment. 

I'm trying to reproduce this [notebook](https://github.com/xtyangpsp/SeisGo/blob/master/notebooks/seisgo_download_xcorr_demo.ipynb). When downloading the data it reports the following 
```
time segment:2014_02_01_00_00_00 to 2014_02_01_06_00_00
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
time segment:2014_02_01_06_00_00 to 2014_02_01_12_00_00
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC05
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
module 'scipy.signal' has no attribute 'hann' for DEC07
```
Could this be related to the previous issue where some scipy.signal functions are now under [scipy.signal.windows](https://docs.scipy.org/doc/scipy-1.13.0/reference/signal.windows.html#module-scipy.signal.windows)?

Thank you

**Comment by xtyangpsp** [comment]
Thanks for reporting the issue. I was able to narrow it down to ObsPy compatibility issue. The error comes from obspy.core.util.base._get_function_from_entry_point() [https://docs.obspy.org/packages/autogen/obspy.core.util.base._get_function_from_entry_point.html]. I have fixed this in `utils.py` by directly using `hann` in taper(), without calling the above ObsPy function. 

The version on GitHub has been updated. No new release on pip yet.

**Comment by xtyangpsp** [comment]
I found another place that uses ObsPy's taper function, which is the cause of the error. I have replaced it with seisgo.utils.taper(), which is independent of ObsPy's taper function. The seisgo version on both GitHub and pip has been consistently updated. I also updated and tested the seisgo_download_xcorr_demo.ipynb with the latest usage. I should run just fine. Let me know if you have any further issues with it. Thanks again for using SeisGo and report errors.
